### PR TITLE
Doha - Terraform initialization

### PR DESCRIPTION
**Pull request by DewDropsDoha**
Fixes #174 

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [ ] New feature
- [ ] Refactor
- [x] Infrastructure
- [ ] Other (please specify)

## What was done
Terraform initialized following this link
https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/aws-get-started

## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: doha-aws-node-project/terraform/s3bucket.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+resource "aws_s3_bucket" "lambda_bucket" {
+  bucket = "doha-terraform-bucket"
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/terraform/s3bucket.tf]: S3 bucket name used from env. Commit a4b1752

File: doha-aws-node-project/terraform/terraform.tfstate [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/terraform/terraform.tfstate:1]: tfstate file included in gitignore. Commit cdafa92

[the remark this answers is not in the document]
Comment (mhazad) [reply doha-aws-node-project/terraform/terraform.tfstate:1]: And this file needs to be deleted from the Git.

**Review by mhazad — CHANGES_REQUESTED** [review]
Pls, remove unnecessary files.

File: doha-aws-node-project/terraform/terraform.tfstate (hunk 2 above)
[the remark this answers is not in the document]
Comment (DewDropsDoha) [reply doha-aws-node-project/terraform/terraform.tfstate]: tfstate file removed. Commit 9e0f54f
yarn.lock file removed. Commit 52db6fa

**Review by mhazad — APPROVED** [review]
LGTM

**Pull request merged**